games-fps/urbanterror/urbanterror-4.3.3_p20180410.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
no need to quote it

Reply:
but it's a string :) Whissi pointed it out to me when I took over PM for this package and was it was somewhat educational for me. It doesn't hurt, so I'd like to keep it here.